libs/platform/src/lib/components/select/select.component.ts

```diff
@@ -0,0 +1,134 @@
+import { 	
+    Component,
+    OnInit,
+    Input,
+    forwardRef,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    ContentChild,
+    TemplateRef,
+    AfterContentInit } from '@angular/core';
+import { NG_VALUE_ACCESSOR } from '@angular/forms';
+import { OptionComponent, SelectComponent as fdSelect } from '@fundamental-ngx/core';
+/**
+ * Interface SelectItem is used to deal with complex object in order to be able to format custom label that is
+ * shown in the options.
+ */
+export interface SelectItem {
+    /**
+     * Item text shown in the popup
+     */
+     label: string;
+
+    /**
+     * References to the object instance
+     */
+    value: any;
+    disabled?: boolean;
+
+    /**
+     * Trigger values is a text for selected item
+     */
+    triggerValue?: string;
+}
+
+
+@Component({
+    selector: 'fdp-select',
+    templateUrl: './select.component.html',
+    styleUrls: ['./select.component.scss'],
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => SelectPlatformComponent),
+            multi: true
+        }],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SelectPlatformComponent extends fdSelect implements OnInit, AfterContentInit  {
```

Comment:
Is there a reason we are "extending" the core SelectComponent? It seems like overkill since we are utilizing the core select component in the template.

Reply:
@KevinOkamoto  it was frank suggestion not to recreate the variables instead overload the parent core class and use the variables in the platform component